doc/adr/template.md

```diff
@@ -0,0 +1,62 @@
+# [ADR-xxxx] <!-- short title of solved problem and solution -->
+
+Status: **[proposed | rejected | accepted | deprecated | … | superseded by [ADR-xxxx](xxxx-example.md)]**
+Deciders: [list everyone involved in the decision] <!-- optional -->
```

Comment:
There's a slight problem here with regard to Sphinx, I'm afraid: Sphinx does not render single linebreaks as a linebreak. So we should either have two linebreaks here or use bullet points.